packages/opentelemetry-plugin-xml-http-request/test/xhr.test.ts

```diff
@@ -128,22 +152,20 @@ describe('xhr', () => {
         let webTracerProviderWithZone: WebTracerProvider;
         let dummySpanExporter: DummySpanExporter;
         let exportSpy: any;
+        let clearResourceTimingsSpy: any;
         let rootSpan: api.Span;
         let spyEntries: any;
-        const url = `${window.location.origin}/xml-http-request.js`;
+        const url = 'http://localhost:8090/xml-http-request.js';
```

Comment:
The link doesn't point to anything in the test suite, it ends up getting called with a FakeXMLHttpRequest so I don't think it would be affected
The tests written for the fetch api do this hard coding as well so I think it should be safe: https://github.com/open-telemetry/opentelemetry-js/blob/a2d1964b16018837c009f6f75351a41a00a66650/packages/opentelemetry-plugin-fetch/test/fetch.test.ts#L100